burn-import/src/burn/node/conv_transpose_2d.rs

```diff
@@ -0,0 +1,50 @@
+use burn::{
+    nn::conv::{ConvTranspose2dConfig, ConvTranspose2dRecord, ConvTranspose2dRecordItem},
+    record::PrecisionSettings,
+};
+use quote::quote;
+
+use crate::burn::{OtherType, TensorType, Type};
+
+use super::{Node, NodeCodegen};
+
+#[derive(new, Debug, Clone)]
+pub struct ConvTranspose2dNode {
+    name: String,
+    input: TensorType,
+    weight: Tensor,
```

Comment:
I have some difficulties here: If weights are predefined in the model we don't have some variable in global scope to bind with. I see two ways to solve this problem:

1. We can create some virtual (?) `ConstantNode` with auto-generated name (we need to fill `name` or it's panic during `Ident` build).
2. We can just codegen this constant during `field_init` (it's require to make `ir::Tensor` visible at this part of the crate).

Example of constant `weight` argument:

```
Argument {
    name: "",
    ty: Tensor(
        TensorType {
            elem_type: Float32,
            dim: 4,
            shape: Some(
                [
                    256,
                    256,
                    4,
                    4,
                ],
            ),
        },
    ),
    value: Some(
        Float32s([-0.0015144815, 0.005401252, -0.0005880306, 0.004933298, -0.0038426989, 0.0032286698, -0.0051461025, ...]),
    ),
    passed: false,
},
```